static/app/views/replays/replayTable/tableCell.tsx

```diff
@@ -615,21 +615,14 @@ export function ActivityCell({replay, showDropdownFilters}: Props) {
   );
 }
 
-export function PlayPauseCell({
-  isSelected,
-  handleClick,
-}: {
-  handleClick: () => void;
-  isSelected: boolean;
-}) {
+export function PlayPauseCell({isSelected}: {isSelected: boolean}) {
   const inner = isSelected ? (
     <ReplayPlayPauseButton size="sm" iconSize="sm" priority="default" borderless />
   ) : (
     <Button
       title={t('Play')}
       aria-label={t('Play')}
       icon={<IconPlay size="sm" />}
-      onClick={handleClick}
```

Comment:
I don't think we want to remove the onClick here, we still want those with screen readers (and tests) to be able to click the button labeled 'Play'.